prompt_toolkit/widgets/base.py

```diff
@@ -508,6 +509,15 @@ def _(event):
         def _(event):
             self.current_value = self.values[self._selected_index][0]
 
+        @kb.add(Keys.Any)
+        def _(event):
+            # We first check values after the selected value, then all values
```

Comment:
(And if you find the time to fix the comment below. Usually I terminate sentences with a dot. ;) It wouldn't keep me from merging it though.)